sbws/core/scanner.py

```diff
@@ -270,11 +270,7 @@ def measure_relay(args, conf, destinations, cb, rl, relay):
 
 
 def dispatch_worker_thread(*a, **kw):
-    try:
-        return measure_relay(*a, **kw)
-    except Exception as err:
-        log.exception('Unhandled exception in worker thread')
-        raise err
+    return measure_relay(*a, **kw)
```

Comment:
Does this stop us from raising an exception? Doesn't it still raise an exception if it triggers?

Reply:
Yes, an exception will be raised if it's triggered. Wrong commit message, i just wanted to remove the log.

Reply:
ACK